packages/documentation/.storybook/styles/preview.scss

```diff
@@ -199,3 +199,9 @@
     right: 0;
   }
 }
+
+.custom-story-container {
+  display: block;
+  margin: -39px -29px;
```

Comment:
I would suggest to set the negative margin to the same values as the posive padding. Or if you prefer this solution, add `border-radius-top-left` and `border-radius-top-right` on the custom-story-container to prevent the strange look like in the following screenshot (open it in its full resolution and inspect the top borders to see what I mean). 
![image](https://github.com/user-attachments/assets/ee1e4b4b-bc40-4f07-be5f-fa65b90c8031)

```suggestion
  margin: -40px -30px;
```